File: src/main/kotlin/ru/blimfy/common/dto/TokenDto.kt
package ru.blimfy.common.dto

/**
 * DTO с токеном авторизации пользователя.
 *
 * @property token токен авторизации.
 * @author Владислав Кузнецов.
 * @since 0.0.1.
 */
data class TokenDto(val token: String)

File: ui/src/api/dto/response/auth/TokenDto.ts
/**
 * DTO с информацией о токене авторизации пользователя.
 *
 * @author Владислав Кузнецов.
 * @since 0.0.1.
 */
// noinspection JSUnusedGlobalSymbols
export class TokenDto {
  /**
   * Токен авторизации пользователя.
   */
  readonly token: string

  /**
   * Инициализирует токен авторизации пользователя.
   *
   * @param token токен пользователя.
   */
  // noinspection JSUnusedGlobalSymbols
  constructor(token: string) {
    this.token = token;
  }
}
